base/challenge-skeleton/README.md

```diff
@@ -0,0 +1,40 @@
+# Quickstart guide to writing a challenge:
+
+* Put your challenge binary under challenge/image/ and update challenge/image/Dockerfile.
+* By default /chroot/home/user/chal will be run. The command line can be changed in challenge/runtime-files/nsjail.cfg.
```

Comment:
We could make it so that the nsjail.cfg doesn't have the command to execute, and instead accept it from the command line. This way the user just has to do in the Dockerfile:
```
CMD drop_privs && nsjail /chroot/chal
```